docs/UserGuide.md

```diff
@@ -1421,4 +1361,4 @@ Output Schema Reference Document](#references). (Booz Allen Hamilton
 # 9 - References
```

Comment:
- Section 8.2, http and https should be sub-bullets under schemas.
- Remove or  update section 8.2.2